File: PRAKTIKUM MODUL 5/PRAK502-2310817220008-AthayaLailySyafitri.py
def hitung(nilai1, nilai2):
    return abs(nilai1 - nilai2)

def mutlak(angka):
    return abs(angka)

x1, y1, x2, y2 = map(int, input().split())

hasil = hitung(x1, x2) + hitung(y1, y2)

print(mutlak(hasil))

File: PRAKTIKUM MODUL 5/PRAK502-2310817220008-AthayaLailySyafitri.c
#include <stdio.h>
#include <math.h>

int hitung(int nilai1, int nilai2) {
    return mutlak(nilai1 - nilai2);
}

int mutlak(int angka) {
    return (angka < 0) ? -angka : angka;
}

int main() {
    int x1, y1, x2, y2;

    scanf("%d %d %d %d", &x1, &y1, &x2, &y2);

    int hasil = hitung(x1, x2) + hitung(y1, y2);

    printf("%d", mutlak(hasil));

    return 0;
}
